### PR TITLE
UIDATIMP-1148: Change the status message for import jobs that are stopped by users

### PR DESCRIPTION
**Pull request by Javokhir12**
## Purpose

- Change the status message for import jobs that are stopped by users

## Approach

- Check for `CANCELLED` status from job.

## Refs

- https://issues.folio.org/browse/UIDATIMP-1148
## Screenshots

https://user-images.githubusercontent.com/89512426/167127069-4869bcf2-d4c7-43f4-8381-fd73f64070b1.mp4

### CODE REVIEW

File: src/components/JobLogsContainer/JobLogsContainer.js [hunk 1]
```diff
@@ -10,12 +10,12 @@ import { listTemplate } from '../ListTemplate';
 import {
   DEFAULT_JOB_LOG_COLUMNS,
   DEFAULT_JOB_LOG_COLUMNS_WIDTHS,
-  FILE_STATUSES,
   checkboxListShape,
   getJobLogsListColumnMapping,
 } from '../../utils';
 
 import sharedCss from '../../shared.css';
+import { statusCellFormatter } from '../../utils/formatStatusCell';
```
Comment (mariia-aloshyna) [inline src/components/JobLogsContainer/JobLogsContainer.js:18]: Move utils import before styles import, please

You can find the rules here - https://wiki.folio.org/pages/resumedraft.action?draftId=14458784&draftShareId=483fa2fb-615e-4002-a916-c976bd5fb8c6&

File: src/routes/ViewAllLogs/ViewAllLogs.js [hunk 2]
```diff
@@ -48,10 +48,12 @@ import {
 } from './ViewAllLogsUtils';
 
 import sharedCss from '../../shared.css';
+import { statusCellFormatter } from '../../utils/formatStatusCell';
```
Comment (mariia-aloshyna) [inline src/routes/ViewAllLogs/ViewAllLogs.js:51]: Same here

File: src/routes/ViewAllLogs/ViewAllLogs.js [hunk 3]
```diff
@@ -284,7 +270,7 @@ class ViewAllLogs extends Component {
         selectedRecords,
       }),
       fileName: fileNameCellFormatter,
-      status: statusCellFormatter,
+      status: statusCellFormatter(formatMessage),
```
Comment (mariia-aloshyna) [inline src/routes/ViewAllLogs/ViewAllLogs.js:273]: Good approach, I like it!

File: src/utils/formatStatusCell.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { FILE_STATUSES } from './constants';
+
+export function statusCellFormatter(formatter) {
```
Comment (mariia-aloshyna) [inline src/utils/formatStatusCell.js:3]: Add documentation for this util, please

File: src/routes/ViewAllLogs/ViewAllLogs.js (hunk 3 above)
Comment (Javokhir12) [reply src/routes/ViewAllLogs/ViewAllLogs.js:273]: thanks

**Review by mariia-aloshyna — APPROVED** [review]

**Review by jakhongirabdukhamdiov — APPROVED** [review]

**Pull request merged**